python-sql-scripts/credentials.py

```diff
@@ -1,5 +1,5 @@
 # MySQL DB Connection Config Values
 mysql_endpoint='localhost'
 username='root'
-password='<Put MySQL Server Password Here>'
+password='ally2009'
```

Comment:
In the future, don't push credentials info to your branch.

In your local repo, use git stash push <credentials.py file path> to make the password go away while you are doing git add . and git commit on your changes. Then, just do git stash pop when you want that password back.

You can do the same with the application.properties file path.